### PR TITLE
Add CSS / JS extensions support

### PR DESCRIPTION
**Pull request by cripplet**
Manually tested. Let me know if there is any formal testing procedure.

PR for https://github.com/Lednerb/bilberry-hugo-theme/issues/52.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,6 +210,12 @@ Follow the steps 1-7 to implement your custom font to your site.
 _Note:_ You don't have to upload the `node_modules` folder to your webspace, this is only created and needed to generate your customized stylesheets.
 
 
+# CSS and JS modules
+This theme supports hot-swappable CSS and JavaSript extentions. Modules can be specified using the `(css|js)_modules` list parameter. Modules can be specified either relative to the `static` directory (e.g. `exampleSite/static/css/custom.css`) or as a URL.
```
Comment (Lednerb) [inline README.md]: _JavaScript_ typo.

File: exampleSite/config.toml [hunk 2]
```diff
@@ -17,6 +17,13 @@ disqusShortname = "bilberry-hugo-theme"
   # Favicon for the browser's favorites and tabs
   favicon = "" # set a favicon icon
 
+  # Custom JS / CSS modules that will be imported by the template.
```
Comment (Lednerb) [inline exampleSite/config.toml]: Please change to "CSS / JS" for consistency.

**Comment by Lednerb** [comment]
Hey @cripplet,
thank you for your contribution!

2 little changes and I will merge this into the `master` branch :+1:

**Comment by cripplet** [comment]
Done.

**Pull request merged**